pkg/datastore/test/datastore.go

```diff
@@ -56,6 +56,11 @@ func AllExceptWatch(t *testing.T, tester DatastoreTester) {
 	t.Run("TestMultipleReadsInRWT", func(t *testing.T) { MultipleReadsInRWTTest(t, tester) })
 	t.Run("TestConcurrentWriteSerialization", func(t *testing.T) { ConcurrentWriteSerializationTest(t, tester) })
 
+	t.Run("TestOrdering", func(t *testing.T) { OrderingTest(t, tester) })
+	t.Run("TestLimit", func(t *testing.T) { LimitTest(t, tester) })
+	t.Run("TestOrderedLimit", func(t *testing.T) { OrderedLimitTest(t, tester) })
+	t.Run("TestResume", func(t *testing.T) { ResumeTest(t, tester) })
```

Comment:
Might be better to be specific here: `TestOrderedQueryRels`, `TestResumeQueryRels`, etc